deployment/helm_chart/opik/values.yaml

```diff
@@ -69,6 +71,7 @@ component:
       AWS_ACCESS_KEY_ID: THAAIOSFODNN7EXAMPLE # rootUser for minio
       AWS_SECRET_ACCESS_KEY: LESlrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY # rootPassword for minio
       S3_URL: "http://opik-minio:9000"
+      CREATE_DEMO_DATA: "true"
```

Comment:
This shouldn't be here; this should affect `demo-data-generator`, which uses the `python-backend`.